lens/registry.go

```diff
@@ -84,24 +84,30 @@ const DefaultPoolSize int = 5
 // NewRegistry instantiates a new registery.
 //
 // It will be of size 5 (per schema version) if a size is not provided.
-func NewRegistry(lensPoolSize immutable.Option[int], db TxnSource) client.LensRegistry {
-	var size int
-	if lensPoolSize.HasValue() {
-		size = lensPoolSize.Value()
-	} else {
-		size = DefaultPoolSize
+func NewRegistry(
```

Comment:
praise: The new param order makes a lot more sense, thanks for making `db` the first one :)